pgml-extension/src/api.rs

```diff
@@ -568,6 +568,15 @@ pub fn embed(transformer: &str, text: &str, kwargs: default!(JsonB, "'{}'")) ->
     crate::bindings::transformers::embed(transformer, text, &kwargs.0)
 }
 
+#[pg_extern(immutable, parallel_safe, name = "embed")]
+pub fn embed_batch(
+    transformer: &str,
+    inputs: default!(Vec<String>, "ARRAY[]::TEXT[]"),
+    kwargs: default!(JsonB, "'{}'"),
+) -> Vec<Vec<f32>> {
+    crate::bindings::transformers::embed_batch(transformer, &inputs, &kwargs.0)
```

Comment:
I would make `crate::bindings::transformers::embed` take a list of inputs always, and modify `pub fn embed` to pass a slice in, similar to `generate` and `generate_batch`.

Reply:
Sure, I'll change this. Makes sense

Reply:
I guess if we do pass an input array always, I could get rid of the try catch since it's always an array of strings.